examples/07-EMIT/ComputeChannelSeparation.py

```diff
@@ -0,0 +1,370 @@
+"""
+EMIT: Computes required channel separation
+---------------------------
+This example shows how you can use PyAEDT to open an AEDT project with
+an EMIT design, and analyze the results to determine the required channel
+separation for overlapping bands. 
+"""
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports.
+import os
+import sys
+import time
+import subprocess
+
+# Check to see which python libraries have been installed
+reqs = subprocess.check_output([sys.executable, '-m', 'pip', 'freeze'])
+installed_packages = [r.decode().split('==')[0] for r in reqs.split()]
+
+# Install required packages if they are not installed
+def install(package): 
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install', package])
+
+# Install any missing libraries
+required_packages = ['tk', 'plotly', 'tqdm', 'matplotlib', 'numpy']
+for package in required_packages:
+    if package not in installed_packages:
+        install(package)
+
+# Import required modules
+from tkinter import Tk
+from tkinter.filedialog import askdirectory
+import plotly.graph_objects as go
+
+sys.path.append("D:\\AnsysDev\\workspace3\\build_output\\64Release\\Delcross")
```

Comment:
I am assuming we can delete this one as we get `Delcross` below using `%ANSYSEM_ROOT231%`
```suggestion
```